### PR TITLE
Compose

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This updates the notebooks by using Compose for generating labels.

### CODE REVIEW

File: Advanced Featuretools RUL.ipynb [hunk 1]
```diff
@@ -697,319 +832,303 @@
...
-    "\n",
-    "from tsfresh.feature_extraction.feature_calculators import (number_peaks, mean_abs_change, \n",
-    "                                                            cid_ce, last_location_of_maximum, length)\n",
-    "\n",
-    "\n",
-    "Complexity = make_agg_primitive(lambda x: cid_ce(x, False),\n",
-    "                              input_types=[vtypes.Numeric],\n",
-    "                              return_type=vtypes.Numeric,\n",
-    "                              name=\"complexity\")\n",
+    "from featuretools_tsfresh_primitives import CidCe\n",
```
Comment (jeff-hernandez) [inline Advanced Featuretools RUL.ipynb]: After installing tsfresh primitives using this command:

```
python -m pip install featuretools[tsfresh]
```

I wasn't able to import the tsfresh primitives from the featuretools package.

```
from featuretools.tsfresh import CidCe
```

Comment (jeff-hernandez) [reply Advanced Featuretools RUL.ipynb]: updated notebooks to import tsfresh primitives from featuretools

File: Advanced Featuretools RUL.ipynb [hunk 2]
```diff
@@ -10,17 +10,17 @@
     "    <img style=\"display:inline\" width=15% src=\"https://upload.wikimedia.org/wikipedia/commons/e/e5/NASA_logo.svg\" alt=\"NASA\" />\n",
     "</p>\n",
     "\n",
-    "This notebook has a more advanced workflow than [the other notebook](Simple%20Featuretools%20RUL%20Demo.ipynb) for predicting Remaining Useful Life (RUL). If you are a new to either this dataset or Featuretools, I would recommend reading the other notebook first. \n",
+    "This notebook has a more advanced workflow than [the other notebook](Simple%20Featuretools%20RUL%20Demo.ipynb) for predicting Remaining Useful Life (RUL). If you are a new to either this dataset or Featuretools, I would recommend reading the other notebook first.\n",
     "\n",
     "## Highlights\n",
     "* Demonstrate how novel entityset structures improve predictive accuracy\n",
-    "* Build custom primitives using time-series functions from [tsfresh](https://github.com/blue-yonder/tsfresh)\n",
+    "* Using TSFresh Primitives from a featuretools [addon](https://docs.featuretools.com/getting_started/install.html#add-ons)\n",
```
Comment (rwedge) [inline Advanced Featuretools RUL.ipynb]: We should try to match the tense of the other bullet points here.  Maybe "Build TSFresh primitive using . . ." or "Use TSFresh primtiives from a . . ."

File: Simple Featuretools RUL Demo.ipynb [hunk 3]
```diff
@@ -355,83 +412,121 @@
...
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lines_to_next_cell": 2
+   },
    "source": [
-    "Let's walk through a row of the `cutoff_times` dataframe. In the third row, we have engine number 3. At 3:20 on January 6, the remaining useful life of engine number 3 is 213. Having a dataframe in this format tells Featuretools that the feature vector for engine number 3 should only be calculated with data from before that point in time. \n",
+    "Let's walk through a row of the `labels_times` dataframe. In the third row, we have engine number 3. At 00:00 on January 6, the remaining useful life of engine number 3 is 206. Having a dataframe in this format tells Featuretools that the feature vector for engine number 3 should only be calculated with data from before that point in time.\n",
```
Comment (rwedge) [inline Simple Featuretools RUL Demo.ipynb:491]: Is the engine number not deterministic?  In the old example engine 3 has 213 RUL several hours after engine 3 in the updated notebook has 206 RUL.

File: Simple Featuretools RUL Demo.ipynb [hunk 4]
```diff
@@ -1164,21 +1263,14 @@
...
     "# Save output files\n",
-    "\n",
-    "import os\n",
-    "\n",
-    "try:\n",
-    "    os.mkdir(\"output\")\n",
-    "except:\n",
-    "    pass\n",
-    "\n",
+    "os.makedirs('output', exist_ok=True)\n",
```
Comment (rwedge) [inline Simple Featuretools RUL Demo.ipynb:1271]: This only works in Python 3.  Does this demo require python 3?

File: Simple Featuretools RUL Demo.ipynb (hunk 3 above)
Comment (jeff-hernandez) [reply Simple Featuretools RUL Demo.ipynb:491]: Correct, the previous example didn't have deterministic label times. We also wanted to adjust the prediction problem to resemble the example in Compose.

File: Advanced Featuretools RUL.ipynb (hunk 2 above)
Comment (jeff-hernandez) [reply Advanced Featuretools RUL.ipynb]: updated bullet point

File: Simple Featuretools RUL Demo.ipynb (hunk 4 above)
Comment (jeff-hernandez) [reply Simple Featuretools RUL Demo.ipynb:1271]: Yes, support for Compose starts on 3.5.

File: README.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ We'll demonstrate an end-to-end workflow using a [Turbofan Engine Degradation Si
 2. [Generate features](#Step-2:-DFS-and-Creating-a-Model)
 3. [Make predictions with Machine Learning](#Step-3:-Using-the-Model)
 
-*To run the notebooks, you need to download the data yourself. Download and unzip the file from[https://ti.arc.nasa.gov/c/13/](https://ti.arc.nasa.gov/c/6/) and place the files in the 'data' directory*
+*To run the notebooks, you need to download the data yourself. Download and unzip the file from [https://ti.arc.nasa.gov/c/13/](https://ti.arc.nasa.gov/c/6/). Then create a 'data' directory and place the files in the 'data' directory.*
```
Comment (rwedge) [inline README.md]: This link title and URL point to different locations

Comment (thehomebrewnerd) [reply README.md]: Fixed

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**